src/main/java/net/minecraftforge/event/ForgeEventFactory.java

```diff
@@ -206,11 +230,17 @@ public static boolean canEntitySpawnSpawner(EntityLiving entity, World world, fl
         }
     }
 
+    @Deprecated
```

Comment:
This should be
`@Deprecated // TODO remove in 1.13. use the MobSpawnerBaseLogic version below.`